.github/workflows/ci.yml

```diff
@@ -18,12 +18,12 @@ jobs:
       - name: Install Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 10
+          node-version: 20
           cache: npm
           cache-dependency-path: 'package.json'
 
       - name: Install dependencies
-        run: npm install
+        run: npm install --legacy-peer-deps
```

Comment:
The `--legacy-peer-deps` option is required for *grunt-template-jasmine-istanbul* v0.6.0, since it depends on *grunt-contrib-jasmine* v2.x and we are now on v4.x. It’s less than ideal, but is only needed in order to install dev dependencies; a consumer of loglevel should not need it when running `npm install loglevel` in their package.

As noted in the PR description, I actually think the best solution if we want to remove this is to drop *grunt-template-jasmine-istanbul* altogether.

Reply:
This sounds like something that would make contribution difficult though, it would be nice if this worked out of the box so that contributors can just run `npm install` without problems.

You can run `npm config --location=project set legacy-peer-deps=true` to fix this - that'll create a `.npmrc` that will enable this setting by default for any npm command run in this repo, and then you can drop that option from here.

(But as noted in my other comment - I think we should just drop this instead, so you don't actually need to do that unless there's some other legacy peer deps issue elsewhere too)

Reply:
Ah, I had not thought about adding a `.npmrc` file for this, good catch! But also, yeah, agree on the downsides anyway.